Cargo.lock

```diff
@@ -18,7 +18,16 @@ version = "0.19.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a76fd60b23679b7d19bd066031410fb7e458ccc5e958eb5c325888ce4baedc97"
 dependencies = [
- "gimli",
```

Comment:
Not sure why this `Cargo.lock` has so many changes, can you try copying the one from master and compiling again? I do it like this:

```
git checkout origin/master -- Cargo.lock
cargo test
git commit -am 'Update Cargo.lock'
```

Reply:
It does not work, unfortunately.

Reply:
Worked for me, fixed